### ISSUE TITLE
Platypus: Utility panel's "Lesson Notes" dropdown missing child roles

### ISSUE THREAD

**Issue by eddybrando** [Qiskit/platypus#477]
According to an Axe a11y audit performed on `/course/ch-prerequisites/environment-setup-guide-to-work-with-qiskit-textbook`, the Utility panel's "Lesson Notes" dropdown contains a `listbox` that is missing child roles according to WCAG 1.3.1: https://dequeuniversity.com/rules/axe/4.1/aria-required-children

This might be tricky to solve as it seems to be an issue with the Carbon dropdown component, which according to its [accessibility section](https://www.carbondesignsystem.com/components/dropdown/accessibility/) has some accessibility issues.

We should still try and see if something can be done from our end to solve this our maybe submit a PR to Carbon to solve this if it's an upstream issue.

---

https://dequeuniversity.com/rules/axe/4.1/aria-required-children

---

**User impact:** Critical
**Disabilities affected:** Blind, Deafblind, Mobility
**Requirement(s):** WCAG 2.0 (A): MUST
**WCAG Success Criteria:** 1.3.1 Info and Relationships

**Comment by vabarbosa** [comment]
This repository is no longer maintained. It has been superseded by [IBM Quantum Learning](https://learning.quantum.ibm.com).
